src/feed/Feed.ts

```diff
@@ -266,10 +272,19 @@ class Feed {
             getHistoryOnly = true;
         }
         if (getHistoryOnly) {
-            const response: TicksHistoryResponse = await this._binaryApi.getTickHistory(
-                tickHistoryRequest as TCreateTickHistoryParams
-            );
-            quotes = TickHistoryFormatter.formatHistory(response);
+            if (this.shouldFetchTickHistory) {
+                // Passed all_ticks from Deriv-app store modules.contract_replay.contract_store.contract_info.audit_details.all_ticks
+                const allTicksContract = await this.allTicks;
+                quotes = TickHistoryFormatter.formatAllTicks(
+                    allTicksContract as keyof AuditDetailsForExpiredContract | []
+                    );
+                
+            } else {
```

Comment:
it doesn't fetch tick history here

Reply:
so, maybe better rename shouldFetchTickHistory to something else in the order it became more logically sense?
or boolean value of shouldFetchTickHistory inverse?

Reply:
I'll edit it to make it more logical

Reply:
fixed

Reply:
pls, check the full code, because here show without refresh 
when `shouldFetchTickHistory` = true, should be call fetch of `ticks_history`